src/Agent.Worker/ContainerOperationProvider.cs

```diff
@@ -708,7 +708,14 @@ private async Task StartContainerAsync(IExecutionContext executionContext, Conta
                     Func<string, string, string, string> addUserWithIdAndGroup;
                     Func<string, string, string> addUserToGroup;
 
-                    if (isAlpineBasedImage)
+                    bool userIdIsLarge = Int64.Parse(container.CurrentUserId) > 256000;
```

Comment:
Can be some constant defined with meaningful name?

Reply:
Fixed!